docs/get-started/alerting.md

```diff
@@ -6,64 +6,161 @@
+3. Enter a contact point name, and add the email addresses for the recipients of the email notifications.
+4. Expand **Optional settings** and fill in any other relevant settings:
+    - Enable the **Single email** option to send a single email to the recipients containing alerts that are firing. For example, if an alert fires for three nodes, this would send only one email listing all three alerts.
+    - Add an optional message to include with the email notifications.
+    - Edit the email subject for the notifications. The default subject line uses the following format [FIRING:*number of alerts firing for the alert rule*](*Name of alert rule and instance*).
+5. If you do not want to be notified when an alert resolves, expand **Notification settings**, and tick the **Disable Resolved Message** checkbox.
+6. If you want your contact point to notify via multiple channels, for example both via Email and Teams, click **New contact point type** and fill out additional contact point type details.
+7. Click the **Test** button to send a test email and make sure your contact point works as expected.
+8. Click the **Save contact point** button at the bottom of the page. The contact point is now listed under **Alerting > Contact points**.
+
+#### Create additional contact points
+
+In addition to Email contact points, you can add a variety of other contact points, including Slack, email, webhooks, PagerDuty, and more.
+
+Follow the steps above to create additional contact points. Different contact points require different configuration information. For example, for Slack, PMM requires the recipient information, the API token and the webhook URL, which you can get from your Slack administrator.
 
 ### Notification policies
-Notification policies determine how Grafana alerts are routed to contact points by setting where, when, and how to send notifications. 
+
+Notification policies determine how Grafana alerts are routed to contact points by defining where, when, and how to send notifications.
 
 For example, you might specify a limit for the number of times a notification is sent during a certain period. This helps ensure that you don't spam your Slack channel with too many notifications about the same issue.
 
+#### PMM Root Notification policy
+
+Percona Alerting comes pre-configured with a Notification Root Policy, which is the default notification policy that PMM applies to all the alerts that don’t have a custom notification policy assigned to them. This is the **grafana-default-email** policy, which uses Email as the default contact point.
+
+#### How matching works
+
+Policies can have one or more child policies. An alert matches if the alert’s labels match all the **Matching Labels** specified on the policy.
+
+Alerts are first verified against the main policy, then against any matching child policies if the **Continue matching subsequent sibling nodes** is enabled.
```

Comment:
"Alerts are first verified against the **main** policy," We need to be careful here, as when I first read it I thought 'main policy' was referring to the ROOT policy...
suggested change: "Alerts are first verified against the PARENT policy,..."